fip-0023.md

```diff
@@ -0,0 +1,46 @@
+---
+fip: X
```

Comment:
This is needed, so let's go ahead and give a FIP number. 23 is next in line so feel free to put that in here. Thanks!